app/scripts/router.js

```diff
@@ -291,8 +293,10 @@ function (
     },
 
     onAnchorClick: function (event) {
+      /*eslint complexity: [2, 7] */
       // if someone killed this event, or the user is holding a modifier
       // key, ignore the event.
+      var self = this;
```

Comment:
oops, must have been left from the time i used a function to test it :D fixed now @vladikoff